packages/schematics/angular/ng-new/index.ts

```diff
@@ -65,6 +65,9 @@ export default function (options: NgNewOptions): Rule {
       apply(empty(), [
         schematic('workspace', workspaceOptions),
         options.createApplication ? schematic('application', applicationOptions) : noop,
+        schematic('ai-config', {
+          tool: options.aiConfig?.length ? options.aiConfig : [AiConfig.None],
```

Comment:
This will cause the prompt not to be ever displayed as a value is always provided.

```suggestion
            tool: options.aiConfig?.length ? options.aiConfig : undefined,
```